optaplanner-docs/src/main/asciidoc/Chapter-Quick_start/Chapter-Quick_start.adoc

```diff
@@ -71,160 +69,27 @@ image::Chapter-Quick_start/cloudOptimizationValueProposition.png[align="center"]
 [[cloudBalancingDomainModelDesign]]
 === Domain Model Design
 
-Beginning with the domain model:
 
+Using a <<domainModelingGuide,domain model>> helps determine which classes are planning entities and which of their properties are planning variables. Using a domain model helps to simplify contraints, improve performance, and increase flexibility for future needs. 
+
+//procedure from chapter 17 - Design Patterns
```

Comment:
that's a subset of the procedure. No need to have the comment if the section above already links it.

This chapter definitely doesn't need the entire procedure - as this is a getting started quickly.
I was first not sure if it needs any of the procedure - we can maybe just present the solution and link to the procedure in chapter 17? OTH, I also see the value of keeping a subset procedure here - and especially asking the key question "what changes during planning"? So I agree that having a short subset procedure here has value.